PlanetNineGateway/iOS/Models/Network.swift

```diff
@@ -122,22 +123,34 @@ class Network: NSObject {
     }
     
     func getUserByUUID(userUUID: String, gatewayName: String, timestamp: String, signature: String, callback: @escaping (Error?, Data?) -> Void) {
+<<<<<<< HEAD:PlanetNineGateway/iOS/Models/Network.swift
+        let path = "/gateway/\(gatewayName)/userUUID/\(userUUID)/signature/\(signature)/timestamp/\(timestamp)"
+=======
         //let path = "/user/userUUID/\(userUUID)/gateway/\(gatewayName)/signature/\(signature)/timestamp/\(timestamp)"
         let path = "/user/\(gatewayName)/userUUID/\(userUUID)/signature/\(signature)/timestamp/\(timestamp)"
+>>>>>>> 54d724ec0e6d892a1ec048e5d0321dc66199a470:PlanetNineGateway/Common/Utilities/Network.swift
```

Comment:
Looks like your PR is conflicting here. You can resolve these pretty easily by removing the bottom and keeping the top it looks like.